aloscene/tensors/spatial_augmented_tensor.py

```diff
@@ -475,6 +475,9 @@ def _resize(self, size, **kwargs):
         h = self.relative_to_absolute(size[0], "h", assert_integer=False)
         # assert_integer = False to avoid raising Assertion. Ex: when coordinates = 900.0000000000001
         w = self.relative_to_absolute(size[1], "w", assert_integer=False)
+        if ("N" in self.names and self.size("N") == 0) or ("C" in self.names and self.size("C") == 0):
```

Comment:
Can you add some comment above the condition to explain slightly what is done in this condition ?

Reply:
Done